<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="jenkins-agent-10-25-196-136" failures="0" tests="6" name="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" time="60.354" errors="0" timestamp="2023-05-17T06:50:52 UTC" skipped="0">
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_429_And_No_MaxRetryTime" time="0.054"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_503" time="40.061"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_502" time="13.056"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_429" time="5.080"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_504" time="2.063"/>
  <system-out/>
  <testcase classname="com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest" name="invokeAsyncTestWith_504_And_No_MaxRetryTime" time="0.040"/>
  <system-out/>
</testsuite> <!-- com.mypurecloud.sdk.v2.ApiClientRetryAsyncTest -->
